app/lib/utils.js

```diff
@@ -0,0 +1,139 @@
+var currentBlob
+
+const baseUrl = $ENVSTATIC_BASE_URL !== "//undefined" ? $ENVSTATIC_BASE_URL : ""
```

Comment:
@almereyda this breaks the tests. if you run ```npm test``` you get:


```
acorbi@acorbi-ThinkPad-X240 ~/Developing/Transformap/transformap-editor (impl-mocking-libs) $ npm test

> transformap-editor@0.0.2 test /media/acorbi/DEV/Developing/Transformap/transformap-editor
> mocha app/test

/media/acorbi/DEV/Developing/Transformap/transformap-editor/app/lib/utils.js:3
const baseUrl = $ENVSTATIC_BASE_URL !== "//undefined" ? $ENVSTATIC_BASE_URL : ""
                ^

ReferenceError: $ENVSTATIC_BASE_URL is not defined
    at Object.<anonymous> (/media/acorbi/DEV/Developing/Transformap/transformap-editor/app/lib/utils.js:3:17)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/media/acorbi/DEV/Developing/Transformap/transformap-editor/app/lib/data_api.js:13:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/media/acorbi/DEV/Developing/Transformap/transformap-editor/app/test/data_api.tests.js:2:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /media/acorbi/DEV/Developing/Transformap/transformap-editor/node_modules/mocha/lib/mocha.js:230:27
    at Array.forEach (native)
    at Mocha.loadFiles (/media/acorbi/DEV/Developing/Transformap/transformap-editor/node_modules/mocha/lib/mocha.js:227:14)
    at Mocha.run (/media/acorbi/DEV/Developing/Transformap/transformap-editor/node_modules/mocha/lib/mocha.js:513:10)
    at Object.<anonymous> (/media/acorbi/DEV/Developing/Transformap/transformap-editor/node_modules/mocha/bin/_mocha:480:18)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:134:18)
    at node.js:961:3
npm ERR! Test failed.  See above for more details.
acorbi@acorbi-ThinkPad-X240 ~/Developing/Transformap/transformap-editor (impl-mocking-libs) $ 
```

can you help fixing?

Reply:
The [brunch documentation on testing](http://brunch.io/docs/testing) says unit tests should currently be run in the browser, and it is currently not possible to run them on `node`.

Another approach they mention is to run the test against the `public/` bundle.

Reply:
@almereyda I do not understand why it is "not possible". Using mocha I should be able to Unit Test the libraries I wrote by requiring them and testing the expecte output of the exported functions.

As you can see on https://github.com/TransforMap/transformap-editor/pull/42/commits/5ce1be93cbd1bd8d9d1421f9ae51d6af335c8455 and https://stackoverflow.com/questions/32147449/mocha-fails-with-variable-not-defined I basically am setting now ```global.$ENVSTATIC_BASE_URL``` on the test files hence fixing the issue.

Reply:
I just need to exclude the tests from the bundle created by brunch